### PR TITLE
PR template added

### PR DESCRIPTION
**Pull request by Anze1m**

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Checklist:
+
+- [ ] Code follows the style guidelines
+- [ ] Documentation updated
+- [ ] Corner cases evaluated and handled
+- [ ] index.js exports updated
+- [ ] Automatic tests added/update
```
Comment (krzysztof-jelski) [inline .github/pull_request_template.md]: `- [ ] Automatic tests added/updated`

Comment (Anze1m) [inline .github/pull_request_template.md]:
```suggestion
 - [ ] Automatic tests added/updated
```

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Checklist:
+
+- [ ] Code follows the style guidelines
+- [ ] Documentation updated
+- [ ] Corner cases evaluated and handled
+- [ ] index.js exports updated
+ - [ ] Automatic tests added/updated
```
Comment (Anze1m) [inline .github/pull_request_template.md]:
```suggestion
 - [ ] Automatic tests added/updated
```

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# Checklist:
+
+- [ ] Code follows the style guidelines
+- [ ] Documentation updated
+- [ ] Corner cases evaluated and handled
+- [ ] index.js exports updated
+ - [ ] Automatic tests added/updatedfsdfsfsd
```
Comment (Anze1m) [inline .github/pull_request_template.md]:
```suggestion
- [ ] Automatic tests added/updated
```
